README.md

```diff
@@ -151,12 +151,15 @@ To build logger locally in your workstation, run the following command
     1. `g++ test.cc -o test.exe`
     
 
-## Build logger in docker
+## Build logger in docker local docker container
```

Comment:
What is this line saying?

Reply:
That it is building the logger tester app in a local docker container